extensions/statsd-emitter/src/main/java/io/druid/emitter/statsd/StatsDEmitter.java

```diff
@@ -0,0 +1,96 @@
+package io.druid.emitter.statsd;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSortedSet;
+import com.metamx.common.logger.Logger;
+import com.metamx.emitter.core.Emitter;
+import com.metamx.emitter.core.Event;
+import com.metamx.emitter.service.ServiceMetricEvent;
+import com.timgroup.statsd.NonBlockingStatsDClient;
+import com.timgroup.statsd.StatsDClient;
+
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * Created by michael.schiff on 2/5/16.
+ */
+public class StatsDEmitter implements Emitter
+{
+
+  private final static Logger log = new Logger(StatsDEmitter.class);
+
+  private final StatsDClient statsd;
+  private final Map<String, StatsDEmitterConfig.MetricType> metricTypes;
+
+  public StatsDEmitter(StatsDEmitterConfig config) {
+    statsd = new NonBlockingStatsDClient(config.getPrefix(), config.getHostname(), config.getPort());
```

Comment:
please use this instead https://github.com/tim-group/java-statsd-client/blob/master/src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java#L89 like that you have a handler of exception.